### PR TITLE
Game piece tuning

### PR DESCRIPTION
**Pull request by yetiDS**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -61,7 +61,6 @@ public Robot() {
      */
     @Override
     public void robotInit() {
-
         autoChooser = new SendableChooser<>();
         autoChooser.setDefaultOption(AutoModes.CONE_ONE.name, AutoModes.CONE_ONE);
         autoChooser.addOption(AutoModes.MIDDLE_BALANCE.name, AutoModes.MIDDLE_BALANCE);
```
Comment (Amehta996) [inline src/main/java/frc/robot/Robot.java:66]: looks good

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ public RobotContainer(
     private void configureBindings() {
         buttonHelper.createButton(1, 0, new IntakeRollInCommand(intakeSubsystem, IntakeConstants.INTAKE_SPEED)
                 .alongWith(new ConeInCubeOutCommand(carriageSubsystem)), RunCondition.WHILE_HELD);
-        buttonHelper.createButton(6, 0, new IntakeRollOutCommand(intakeSubsystem, IntakeConstants.INTAKE_SPEED)
+        buttonHelper.createButton(6, 0, new IntakeRollOutCommand(intakeSubsystem, 0.21)
```
Comment (Amehta996) [inline src/main/java/frc/robot/RobotContainer.java:84]: Let's make this a constant

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -102,30 +102,15 @@ private void configureBindings() {
...
         buttonHelper.createButton(6, 1, new GridAlignCommand(drivetrainSubsystem, autoBuilder, AutoConstants.ALIGNMENT_POSITION.RIGHT), RunCondition.WHEN_PRESSED);
         buttonHelper.createButton(2, 1, new GridAlignCommand(drivetrainSubsystem, autoBuilder, AutoConstants.ALIGNMENT_POSITION.MIDDLE), RunCondition.WHEN_PRESSED);
         buttonHelper.createButton(7, 1, new GridAlignCommand(drivetrainSubsystem, autoBuilder, AutoConstants.ALIGNMENT_POSITION.MIDDLE), RunCondition.WHEN_PRESSED);
         buttonHelper.createButton(3, 1, new GridAlignCommand(drivetrainSubsystem, autoBuilder, AutoConstants.ALIGNMENT_POSITION.LEFT), RunCondition.WHEN_PRESSED);
         buttonHelper.createButton(8, 1, new GridAlignCommand(drivetrainSubsystem, autoBuilder, AutoConstants.ALIGNMENT_POSITION.LEFT), RunCondition.WHEN_PRESSED);
+        buttonHelper.createButton(4, 1, new ChuteAlignCommand(drivetrainSubsystem, ledSubsystem, carriageSubsystem, primaryController::getLeftX, AutoConstants.ALIGNMENT_POSITION.SINGLE_STATION), RunCondition.WHEN_PRESSED);
```
Comment (Amehta996) [inline src/main/java/frc/robot/RobotContainer.java:113]: 👍

File: src/main/java/frc/robot/commands/carriage/CarriageFlipOutCommand.java [hunk 4]
```diff
@@ -16,17 +17,16 @@ public CarriageFlipOutCommand(CarriageSubsystem carriageSubsystem) {
...
     public void execute() {
-
+        carriageSubsystem.setSetpoint(CarriageConstants.CarriagePositions.FLIPPED);
     }
 
     @Override
     public boolean isFinished() {
-        return true;
+        return carriageSubsystem.atSetpoint();
     }
```
Comment (Amehta996) [inline src/main/java/frc/robot/commands/carriage/CarriageFlipOutCommand.java:31]: lgtm

File: src/main/java/frc/robot/commands/drive/ChuteAlignCommand.java [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    @Override
+    public boolean isFinished() {
+        return false;
+    }
+
+    @Override
+    public void end(boolean interrupted) {
+        drivetrainSubsystem.stop();
+    }
+}
```
Comment (Amehta996) [inline src/main/java/frc/robot/commands/drive/ChuteAlignCommand.java:119]: lgtm

**Review by Amehta996 — CHANGES_REQUESTED** [review]
Change constant

**Review by Amehta996 — APPROVED** [review]
ready

**Review by Amehta996 — APPROVED** [review]
looks good

**Review by tsar-boomba — APPROVED** [review]
LGTM! 🌈 👍

**Pull request merged**